src/examples/Punctuality/README.md

```diff
@@ -22,12 +23,12 @@ const App = () => {
         key: 'ch.sbb.netzkarte',
         layers: [
           ...defaultBaseLayers,
-          new TrajservLayer({
+          new TralisLayer({
```

Comment:
The example doesn't open the route schedule, in this preview app and on [jsdoc](https://jsdoc.maps.trafimage.ch/?baselayers=ch.sbb.netzkarte,ch.sbb.netzkarte.dark,ch.sbb.netzkarte.luftbild.group,ch.sbb.netzkarte.landeskarte,ch.sbb.netzkarte.landeskarte.grau&lang=de&layers=#/Trafimage%20Maps?id=webcomponent). Is this the expected behaviour?

Reply:
I think yes . It was like that before. The window is too small to display nicely the trackerMenu.